src/legacy/core_plugins/data/public/index.ts

```diff
@@ -69,7 +69,6 @@ export {
   isStringType,
   isType,
   isValidInterval,
-  isValidJson,
```

Comment:
I had moved `isValidJson` into the default editor in #59605 since it is the only place where it is currently being used. Since this PR will likely merge first I'll go back and adjust #59605 accordingly.